index.html

```diff
@@ -3862,6 +3855,15 @@ <h4>updated</h4>
                 </p>
               </section>
 
+          <section>
+            <h4>deactivated</h4>
+            <p>
+This property MUST be populated with a <a data-cite="INFRA#boolean">boolean</a> 
+`true` value if the DID Method determines that the <a>DID</a> supplied has been 
```

Comment:
```suggestion
`true` value if the supplied <a>DID</a> has been 
```

I'd prefer to simplify this sentence a bit, but I'd also be fine with the PR as-is.

Reply:
The WG proposed the following text in a special topic call, there were no objections: This property MUST be populated by a boolean value that is true if the DID has been deactivated, and false if the DID is still active.

Since you were on that call, I'm going to go with that language.